### PR TITLE
Add wireguard vpn section

### PR DESCRIPTION
**Pull request by doanac**
Signed-off-by: Andy Doan <andy@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 97db5e70939e2f54a98264cd0b9a399ea45b5a91 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/437/docs/artifacts/html/index.html

**Comment by mike-scott** [comment]
+1 to merge as is and we'll shuffle this around during the documentation changes

File: source/reference/vpn.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+   $ ./factory-wireguard.py \
+       --apitoken <api token> \  # https://app.foundries.io/settings/tokens
+       --factory <factory> \
+       --privatekey /root/wgpriv.key \ # where to store generated private key
+       enable
+
+
+Run the daemon::
+
+   $ ./factory-wireguard.py \
```
Comment (mike-scott) [inline source/reference/vpn.rst:49]: Does this need sudo permission?

Comment (doanac) [reply source/reference/vpn.rst:49]: it does. i'll update that.

**Comment by doanac** [comment]
Docs for 7be74665b86ed906b89e1958f5ef9f1fbe4f3319 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/438/docs/artifacts/html/index.html

**Pull request merged**